### DISCUSSION TITLE
Question about the use of --trusted-contigs feature in Spades

### DISCUSSION THREAD

**Discussion by Subhajeet1997** [ablab/spades#1369] [Q&A]
Hello,
Hi, I'm attempting to use Spades' "--trusted-contigs" option. I have a draft genome assembly made with Flye assembler for a bacterium isolate. We have sequenced the metagenome of a mixed culture that includes that particular strain of bacteria. However, because the reads mapped to the specific bacteria were quite low, I was unable to retrieve the bin of that particular bacteria when I performed spades de novo assembly using meta and non-meta options, followed by binning using metabat2. Hence, I repeated Spades assembly without the meta option and used the Flye assembled bacterial genome as the trusted contigs, resulting in a mixed assembly. I now have the bin containing the particular bacterium after being binning by metabat2. 
My primary goal in this case is to look for any structural and/or genomic differences between the previous genome and the bin that was taken from the culture. My issue is whether this makes sense because the bin I obtained included sequences from the trusted contigs from the previous genome, therefore those sequences will be present in the bin. Is this correct, or will the bin sequence be determined only by the metagenomic reads and not by the sequences of trusted contigs? I'm confused about these things. Would anyone kindly elaborate? 

Regards,

**Comment by asl** [comment]
I believe the answer could be directly found in manual (https://ablab.github.io/spades/hybrid.html):

> In case you have contigs of the same genome generated by different assembler(s), and you wish to add them into SPAdes assembly, you can specify additional contigs using --trusted-contigs or --untrusted-contigs. The first option is used when high quality contigs are available. These contigs will be used for graph construction, gap closure and repeat resolution. The second option is either used for less reliable contigs, which may have more errors, or for contigs of unknown quality. Such contigs will be used only for gap closure and repeat resolution.

> Note, that SPAdes does not perform assembly using genomes of closely-related species. Only contigs of the same genome should be provided.

Your case is exactly the opposite – your trusted contigs are from different sample and therefore your assembly is a mixture.
